CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [0.6.1] - 2020-09-14
```

Comment:
This should be 0.7.0 as we have `### Changed`.

Reply:
Do you suggest adding another 0.7.0 release?

Reply:
Let's leave like it is. Just a note for the future.